DaS-PC-MPChan/IRCClient.vb

```diff
@@ -53,21 +53,48 @@ Public Class IRCClient
         blackSet.Add(self.SteamId)
 
         Dim candidates As New List(Of DSNode)
+
+        Dim DkmSelectedPref As DkmPref = DSCM.DkmPrefBox.SelectedItem
+
         For Each t In ircNodes.Values
             Dim node As DSNode = t.Item1
+
+            'WIP: Darkmoon proprocessing for Blacklist
+            If DkmSelectedPref.Value <> 0 Then
+                If DSCM.DkmCheckBoxSinnersOnly.Checked Then
+                    If node.Indictments < 1 Then
+                        blackSet.Add(node.SteamId)
```

Comment:
Why do you modify the `blackSet` here, instead of just doing `Continue For`? The `blackSet` is not used outside this loop ;).